service-impl/user-service-impl/hashmap-user-service/src/main/java/uk/gov/gchq/palisade/user/service/impl/HashMapUserService.java

```diff
@@ -49,6 +68,42 @@ public HashMapUserService(final boolean useStatic) {
         }
     }
 
+    @Override
+    public void configure(final InitialConfig config) throws NoConfigException {
+        requireNonNull(config, "config");
+        //extract cache
+        String serialisedCache = config.getOrDefault(CACHE_IMPL_KEY, null);
+        if (nonNull(serialisedCache)) {
+            cacheService = JSONSerialiser.deserialise(serialisedCache.getBytes(JSONSerialiser.UTF8), CacheService.class);
+        } else {
+            throw new NoConfigException("no cache service specified in configuration");
+        }
+    }
+
+    @Override
+    public void writeConfiguration(final InitialConfig config) {
```

Comment:
Again just make it clear by renaming the method that this method is the default configuration specific to this class

Reply:
It is service class specific, but for any configurations not just the default.